cardano-api/cardano-api.cabal

```diff
@@ -146,10 +146,13 @@ library
                       , ouroboros-consensus
                       , ouroboros-consensus-byron
                       , ouroboros-consensus-cardano
+                      , ouroboros-consensus-diffusion
                       , ouroboros-consensus-protocol
                       , ouroboros-consensus-shelley
                       , ouroboros-network
+                      , ouroboros-network-api
                       , ouroboros-network-framework
+                      , ouroboros-network-protocols
```

Comment:
The added packages should have bounds.

Reply:
This is a problem that inflicts lots of packages.  I'm intending to add them in a separate PR and fix it up for all packages.